src/frontends/pytorch/src/op/matmul.cpp

```diff
@@ -0,0 +1,21 @@
+ #include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+using namespace ov::op;
+OutputVector translate_mv(const NodeContext& context) {
+    num_inputs_check(context, 3, 7);  
```

Comment:
```suggestion
    num_inputs_check(context, 2, 3);  
```
Add a check for the `out` parameter https://pytorch.org/docs/stable/generated/torch.mv.html
If present, add a check
```
if (!context.input_is_none(2)) {
    context.mutate_input(2, result);
}
```
at the end of the file, before the return statement